README.md

```diff
@@ -17,3 +17,8 @@ Use [CocoaPods](https://guides.cocoapods.org/using/getting-started.html)!
 ```ruby
 pod "opentracing"
 ```
+
+## License
+
+By contributing to OpenTracing documentation repository, you agree that your contributions will be licensed under its [MIT License](./LICENSE).
```

Comment:
We are actively moving to Apache 2.0 across all OT projects, so I would prefer to do that here, too. Also, this is not the documentation repo.

I would suggest that we swap in the ASL2.0, link to it here, create the PR, then give people 2w to object before merging (since it's a license change). I believe that protocol is fully legal and frankly I'd be shocked if anyone objects. They are quite similar as licenses (but CNCF requires Apache2).